### ISSUE TITLE
Markdown as string not working (remark-gfm warning)

### ISSUE THREAD

**Issue by sandrina-p** [remarkjs/react-markdown#489]
### Subject of the issue

I'm building a React app. When I install `react-markdown` and pass the markdown as a string it gives a console warning:

> [remark-gfm] Warning: please upgrade to remark 13 to use this plugin 
    at ReactMarkdown

I have installed `remark` and it still happens.

```js
import ReactMarkdown from 'react-markdown'
import gfm from 'remark-gfm'

// ...

<ReactMarkdown plugins={[gfm]} children={'#Hello world'}></ReactMarkdown>
```

### Your environment

*   **OS**: macos Catalina
*   **Packages**:   "remark": "^13.0.0", "react-markdown": "^4.3.1", "remark-gfm": "^1.0.0",
*   **Env**: node 12.18.2, npm 6.14.5

### Steps to reproduce

Codesandox: https://codesandbox.io/s/react-markdown-bug-remark-gfm-13-jvi54

### Expected behavior

Not showing a  warning

### Actual behavior

Shows incorrect warning

**Comment by wooorm** [comment]
https://github.com/remarkjs/react-markdown/issues/470

**Comment by wooorm** [comment]
Fixed in 5.0.0!
